avm/res/network/network-manager/main.bicep

```diff
@@ -171,6 +177,19 @@ module networkManager_securityAdminConfigurations 'security-admin-configuration/
   }
 ]
 
+module networkManager_routingConfigurations 'routing-configuration/main.bicep' = [
+  for (routingConfiguration, index) in routingConfigurations ?? []: {
+    name: '${uniqueString(deployment().name, location)}-NetworkManager-RoutingConfigurations-${index}'
+    params: {
+      name: routingConfiguration.name
+      networkManagerName: networkManager.name
+      description: routingConfiguration.?description ?? ''
```

Comment:
Are the fallback defaults here intended? If not provided for a nullable parameter they just won't be used. In other words: I'd only recommend setting a fallback default here if you definitely want to not pass `null`.

Reply:
I've set this as a default value on the param directly instead and allowed nullable here.

Reply:
Neat, thanks. I think this will make it a bit clearer (also because now the user can see the default in the readme). Much appreciated, thanks 💪